src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt

```diff
@@ -37,17 +36,19 @@ class ProtectLevelCalculator(
 
   fun calculateProtectLevel(
     crn: String,
-    offenderAssessment: OffenderAssessment?,
-    rsrScore: BigDecimal,
-    registrations: Collection<Registration>,
-    convictions: Collection<Conviction>
+    offenderAssessment: OffenderAssessment? = null,
+    rsrScore: BigDecimal = BigDecimal.ZERO,
```

Comment:
rsrScore is already defaulted here:
https://github.com/ministryofjustice/hmpps-tier/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/DeliusAssessments.kt#L14
We shouldn't default it again here